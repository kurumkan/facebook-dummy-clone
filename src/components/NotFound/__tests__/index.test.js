import NotFound from '../index.js';
import React from 'react';
import { MemoryRouter } from 'react-router-dom'
import renderer from 'react-test-renderer';

describe('NotFound', () => {
  it('renders correctly', () => {
    const rendered = renderer.create(
      <MemoryRouter>
        <NotFound />
      </MemoryRouter>
    );
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});