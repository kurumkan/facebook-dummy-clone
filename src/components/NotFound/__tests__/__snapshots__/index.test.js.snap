// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders correctly 1`] = `
<section
  className="notfound-section"
>
  <div
    className="notfound-content"
  >
    <h1>
      404
    </h1>
    <h2>
      Sorry, we can't find the page!
    </h2>
    <p>
      Either something went wrong or the page doesn't exist anymore
    </p>
    <a
      className="btn"
      href="/"
      onClick={[Function]}
    >
      Take me home
    </a>
  </div>
</section>
`;
